platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -157,6 +161,27 @@ public void run() {
 		return true;
 	}
 
+	public boolean onTouchEvent(final MotionEvent event) {
```

Comment:
Can you add a javadoc comment explaining what the logic below does.
It doesn't look straightforward, and once the intent is clarified, I suspect there may be better ways to achieve the same goal.